calibration/CTDMOG/CGINS-CTDMOG-13621__20150618.csv

```diff
@@ -1,25 +1,25 @@
 serial,name,value,notes
-37-13621,CC_a0,-0.0001281035,
+37-13621,CC_a0,-0.0001281035,source_file 3305-00101-00214-A_SN_37-13621_QCT_Results_CTDMO-G.cap
 37-13621,CC_a1,0.0003141495,
 37-13621,CC_a2,-5.137526e-06,
 37-13621,CC_a3,2.186788e-07,
 37-13621,CC_cpcor,-9.57e-08,
 37-13621,CC_ctcor,3.25e-06,
-37-13621,CC_g,-0.9801524,
+37-13621,CC_g,-9.801523e-01,
 37-13621,CC_h,0.1433896,
 37-13621,CC_i,-0.0001246696,
 37-13621,CC_j,2.982709e-05,
 37-13621,CC_p_range,1450,
-37-13621,CC_pa0,0.7502893,
+37-13621,CC_pa0,7.502894e-01,
 37-13621,CC_pa1,0.004517037,
 37-13621,CC_pa2,-2.77234e-11,
 37-13621,CC_ptca0,524655.3,
 37-13621,CC_ptca1,5.235418,
-37-13621,CC_ptca2,0.08071017,
+37-13621,CC_ptca2,8.071018e-02,
 37-13621,CC_ptcb0,25.11725,
 37-13621,CC_ptcb1,-0.00035,
 37-13621,CC_ptcb2,0,
 37-13621,CC_ptempa0,155.3476,
 37-13621,CC_ptempa1,-0.0704664,
 37-13621,CC_ptempa2,5.438232e-07,
-37-13621,CC_wbotc,3.823e-07,
+37-13621,CC_wbotc,3.822987e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT